invadedhouse/commands.py

```diff
@@ -145,5 +152,31 @@ def interpret_command(self, cmd: str):
                     print(f'{item_str} is empty - nothing to loot!')
             else:
                 print('Can\'t loot. Try to loot a different loot.')
+        elif cmd == 'smash a wall' or cmd == 'smash the wall':
+            place_that_the_hole_leads_to = self.house.find_room_by_name('outside')
+            room_that_lets_you_into_the_sword_room = self.house.find_room_by_name('entry room')
+            room_with_a_wall_that_can_be_smashed = self.house.find_room_by_name('sword room')
+            if not self.player.get_location() == room_with_a_wall_that_can_be_smashed:
+                print('Smash attempt failed.')
+                return
+            else:
+                print('The wall has been smashed! You walk through your hole in the wall and you are outside.')
+                print('')
+                room_with_a_wall_that_can_be_smashed.set_exits([room_that_lets_you_into_the_sword_room, place_that_the_hole_leads_to])
```

Comment:
Smash a wall!

Reply:
this was also a super super clever command that you added, you're really learning how to control what the user can do and change the state of the game after they do it!

another thing we _could_ do is something like `Room.add_exit` so you don't have lookup the existing exits just to add a new one... but it is SUPER CLEVER that you figured out how to do that!!